inst/rmarkdown/templates/ajs_outbreak/skeleton/skeleton.Rmd

```diff
@@ -1581,126 +1890,160 @@ mortality_rate(deaths$deaths, deaths$population, multiplier = 10000) %>%
-## - [ ] Choose the appropriate section below depending on if you are plotting polygons or points
-## - [ ] Choose what variable you would like to fill (for polygons) or colour (for points) with (counts or AR)
-## - [ ] DELETE OR COMMENT OUT THE SECTION YOU WONT BE USING
+## Create a categories of ARs by region ----------------------------------------
 
-## create a categorical variable for plotting 
-max_ar    <- max(ar$`Upper 95%CI`, na.rm = TRUE) # define your highest AR
+# define maximum
+max_ar    <- max(ar_region$`Upper 95%CI`, na.rm = TRUE) # define your highest AR
 
+# create groups - one group will be 0 only, then up to 4 more groups evenly
+# divided to maximum
 breakers <- as.integer(c(0, # include zero as a standalone group
              seq(1, max_ar, by = max_ar/4) # 1 to maximum with four divisions
              ))
-## add a categorical variable using the age_categories function 
-## nb in this case we arent using ages - but it functions the same way!
-ar <- mutate(ar, 
+
+## create a categorical variable using the age_categories function 
+## (we aren't using ages - but it functions the same way!)
+ar_map <- mutate(ar_region, 
              categories = age_categories(`AR (per 10,000)`, 
-                                         breakers = breakers)
-             )
+                                         breakers = breakers))
+
 
 
-## join your ar or case counts based on matching shapefile names with regions in your ar table
+## Join table to shapefile -----------------------------------------------------
 mapsub <- left_join(map, ar, by = c("name" = "Region"))
 
 
-## plotting with polygons ------------------------------------------------------
 
-## choropleth 
+## Plot for polygons -----------------------------------------------------------
+
 ## you could also fill by cases using `Cases (n)` in the fill option instead of `AR (per 10,000)`
 ggplot() +
-  geom_sf(data = mapsub, aes(fill = categories), col = "grey50") + # shapefile as polygon
-  coord_sf(datum = NA) + # needed to avoid gridlines being drawn
-  annotation_scale() + # add a scalebar
+  # shapefile as polygon
+  geom_sf(data = mapsub, aes(fill = categories), col = "grey50") + 
+  
+  # needed to avoid gridlines being drawn
+  coord_sf(datum = NA) + 
+  
+  # add a scalebar
+  annotation_scale() + 
+  
   # color the scale to be perceptually uniform 
   # drop FALSE keeps all levels 
   # name allows you to change the legend title 
   scale_fill_brewer(drop = FALSE, palette = "OrRd", name = "AR (per 10,000)") + 
-  geom_sf_text(data = mapsub, aes(label = name), colour = "grey50") + # label polygons
-  theme_void() # remove coordinates and axes
+  
+  # label polygons
+  geom_sf_text(data = mapsub, aes(label = name), colour = "grey50") + 
+  
+  # remove coordinates and axes
+  theme_void() 
 
 
 
-## plotting with points --------------------------------------------------------
+## Plot for points -------------------------------------------------------------
```

Comment:
maybe make a seperate chunk - with an explanatory html comment and set eval = FALSE rather than commented out

Reply:
dropped from this section